include/api.php

```diff
@@ -659,7 +665,7 @@ function api_get_user(App $a, $contact_id = null)
 				'geo_enabled' => false,
 				'verified' => false,
 				'statuses_count' => 0,
-				'lang' => '',
+				'language' => '',
```

Comment:
This has to stay "lang" to stay compatible with the Twitter API, see here: https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/user-object.html